packages/react/src/Grid/GridCell.test.tsx

```diff
@@ -6,38 +6,71 @@ import '@testing-library/jest-dom'
 describe('GridCell', () => {
   it('renders', () => {
     const { container } = render(<GridCell />)
-
     const component = container.querySelector(':only-child')
-
     expect(component).toBeInTheDocument()
     expect(component).toBeVisible()
   })
 
   it('renders a design system BEM class name', () => {
     const { container } = render(<GridCell />)
-
     const component = container.querySelector(':only-child')
-
     expect(component).toHaveClass('amsterdam-grid-cell')
   })
 
   it('can have a additional class name', () => {
     const { container } = render(<GridCell className="extra" />)
+    const component = container.querySelector(':only-child')
+    expect(component).toHaveClass('amsterdam-grid-cell extra')
+  })
 
+  it('supports ForwardRef in React', () => {
+    const ref = createRef<HTMLDivElement>()
+    const { container } = render(<GridCell ref={ref} />)
     const component = container.querySelector(':only-child')
+    expect(ref.current).toBe(component)
+  })
 
-    expect(component).toHaveClass('extra')
+  it('generates class names for single number values for start and span', () => {
+    const { container } = render(<GridCell span={4} start={2} />)
+    const component = container.querySelector(':only-child')
+    expect(component).toHaveClass('amsterdam-grid-cell--span-4 amsterdam-grid-cell--start-2')
+  })
 
-    expect(component).toHaveClass('amsterdam-grid-cell')
+  it('generates class names for a single number value for start', () => {
+    const { container } = render(<GridCell span={8} />)
+    const component = container.querySelector(':only-child')
+    expect(component).toHaveClass('amsterdam-grid-cell--span-8')
   })
 
-  it('supports ForwardRef in React', () => {
-    const ref = createRef<HTMLDivElement>()
+  it('generates class names for a single number value for span', () => {
+    const { container } = render(<GridCell start={6} />)
+    const component = container.querySelector(':only-child')
+    expect(component).toHaveClass('amsterdam-grid-cell--start-6')
+  })
 
-    const { container } = render(<GridCell ref={ref} />)
+  it('generates class names for a single number for span and array values for start', () => {
+    const { container } = render(<GridCell span={8} start={{ narrow: 2, medium: 4, wide: 6 }} />)
+    const component = container.querySelector(':only-child')
+    expect(component).toHaveClass(
+      'amsterdam-grid-cell--span-8 amsterdam-grid-cell--start-2 amsterdam-grid-cell--start-4-medium amsterdam-grid-cell--start-6-wide',
+    )
+  })
 
+  it('generates class names for a single number for span and array values for start', () => {
```

Comment:
```suggestion
  it('generates class names for a single number for start and array values for span', () => {
```

Reply:
Resolved in a commit, wanted to keep mentioning the parameters in order.